### PR TITLE
dump translated-content twice

### LINKED ISSUE #2008: Plan for rendering translated content for launch
1. Change `import.js` to dump two 2 roots for translated content: 
 - `translated-content` (the raw KS HTML)
 - `translated-content-rendered`
1. Create the README and `git push` the `translated-content-rendered` to github.com/mdn/translated-content-rendered`
1. Set the github.com/mdn/translated-content-rendered to Archived. 
1. Edit `prod-build.yml` and `stage-build.yml` to build translated content by default (see line 16 in the Yaml file) and change mentions of `translated-content` to `translated-content-rendered` (e.g. `actions/checkout`)
1. Make a build!
1. Git push the `translated-content` (aka. Project Uplift All) to GitHub for ongoing experimentation
1. Update https://github.com/mdn/yari/pull/1673 to default to the `translated-content` repo for its experimentation. 

Note! Check documentation (e.g. .md files) and code comments that mention "translated-content".

### PR DESCRIPTION
**Pull request by peterbe**
Fixes #2008

### CODE REVIEW

**Comment by peterbe** [comment]
I'm going to merge this. I know it managed to create two perfectly sane-looking repos:
* https://github.com/mdn/translated-content-rendered
* https://github.com/mdn/translated-content

which we can start to use today. 
I'm eager to take this new change for a spin in stage as soon as possible. 
So I'm going to go ahead with this.

**Comment by peterbe** [comment]
@fiji-flo Do you think you can look through this tomorrow? I just didn't want to be blocked by timezones. I'm really eager to start getting translated content into stage as soon as possible.

File: import/import.js [hunk 1]
```diff
@@ -1049,19 +1064,32 @@ async function processDocument(
...
       // The Document.archive() is used for genuinely archived content,
-      // but we also treat translated content the same way ultimately.
-      !isArchive
+      // but we also treat translated content the same way kinda.
+      !isArchive,
+      isArchive ? CONTENT_ARCHIVED_ROOT : CONTENT_TRANSLATED_RENDERED_ROOT
     );
+    builtFilePaths.add(path.join(builtFolderPath, "index.html"));
+
+    // If it's not archive, dump it a second time.
```
Comment (fiji-flo) [inline import/import.js:1080]: Is a `!isArchive` check missing here?

Comment (peterbe) [reply import/import.js:1080]: You were right. I fixed that in a second commit after.

**Pull request merged**